proxy/src/main/java/com/wavefront/agent/PushAgent.java

```diff
@@ -315,14 +317,32 @@ protected void startListeners() {
 
     portIterator(traceListenerPorts).forEachRemaining(strPort ->
         startTraceListener(strPort, handlerFactory, compositeSampler));
-    portIterator(traceJaegerListenerPorts).forEachRemaining(strPort ->
-        startTraceJaegerListener(strPort, handlerFactory,
-            new InternalProxyWavefrontClient(handlerFactory, strPort), compositeSampler));
+    portIterator(traceJaegerListenerPorts).forEachRemaining(strPort -> {
+      PreprocessorRuleMetrics ruleMetrics = new PreprocessorRuleMetrics(
+          Metrics.newCounter(new TaggedMetricName("spanSanitization", "count", "port", strPort)),
+          Metrics.newCounter(new TaggedMetricName("spanSanitization", "cpu_nanos", "port", strPort)),
+          Metrics.newCounter(new TaggedMetricName("spanSanitization", "checked-count", "port",
```

Comment:
I think these two can (and should) be null. These metrics are intended to help users identify poorly performing preprocessor rules, and there's nothing user-configurable here, so it can only be more confusing. One of those cases where more metrics doesn't necessarily mean it's better:)

Reply:
Updated cpu nanos and checked count to be null.